### ISSUE TITLE
Bot tries to remove admins

### ISSUE THREAD

**Issue by Adamruns** [ErikBoesen/AntiSpamBot#2]
If an admin of a group tries to send a blocked phrase the bot tries (and fails) to remove the user from the group. It still prints the message saying that it kicked them after deleting the message. To solve this we should only attempt to kick people that are not admin.

**Comment by ErikBoesen** [comment]
In principle I agree with this, but I think it would be much more straightforward to just try a kick, and if it fails then send the error message. It would be a lot more complicated (and use more lambda runtime) to preemptively check who is an admin.

**Comment by Adamruns** [comment]
What if we just check on the message we are currently looking at? Printing an error message would also start to clog the groupme. What I noticed in some of the groups that I implemented this on is something like this . Basically I convinced these guys to add it to the group and they were messing with it. The bot deletes the message and says that it kicked them but in reality it wasn't able to due to permission issues. 
![IMG_2894](https://github.com/ErikBoesen/AntiSpamBot/assets/50643995/f800e085-1289-4468-a8e7-4ff5d50a16f9)
